package.json

```diff
@@ -47,36 +58,31 @@
     "grunt-contrib-clean": "^0.6.0",
     "grunt-eslint": "^17.3.1",
     "grunt-webpack": "^1.0.11",
-    "jest": "^0.1.40",
-    "jest-cli": "^0.7.1",
+    "jest": "^18.1.0",
+    "jest-cli": "^18.1.0",
+    "jest-css-modules": "^1.1.0",
     "postcss-calc": "^5.0.0",
     "postcss-color-function": "^2.0.0",
     "postcss-color-hex-alpha": "^2.0.0",
     "postcss-loader": "^0.7.0",
     "postcss-nested": "^1.0.0",
     "postcss-simple-vars": "^1.1.0",
-    "react": ">=15.0.0",
-    "react-addons-css-transition-group": ">=15.0.0",
-    "react-addons-test-utils": ">=15.0.0",
-    "react-dom": ">=15.0.0",
+    "react": "^15.4.2",
```

Comment:
I think we should leave these as >=15.0.0 that way we don't get conflicts and potentially multiple versions installed.